src/integration-test/tests/citizen/claimantResponse/reject_test.ts

```diff
@@ -0,0 +1,129 @@
+import I = CodeceptJS.I
+import { PartyType } from 'integration-test/data/party-type'
+import { UserSteps } from 'integration-test/tests/citizen/home/steps/user'
+import { ClaimantResponseSteps } from 'integration-test/tests/citizen/claimantResponse/steps/claimant-reponse'
+import { DefendantResponseSteps } from 'integration-test/tests/citizen/claimantResponse/steps/defendant'
+import { ClaimantResponseTestData } from 'integration-test/tests/citizen/claimantResponse/data/ClaimantResponseTestData'
+import { EndToEndTestData } from 'integration-test/tests/citizen/endToEnd/data/EndToEndTestData'
+
+const userSteps: UserSteps = new UserSteps()
+const claimantResponseSteps: ClaimantResponseSteps = new ClaimantResponseSteps()
+const defendantResponseSteps: DefendantResponseSteps = new DefendantResponseSteps()
+
+Feature('Claimant Response').retry(3)
+
+Scenario('As a claimant I can reject the claim @citizen @admissions',
+  async (I: I) => {
+
+    const testData = await EndToEndTestData.prepareData(I, PartyType.INDIVIDUAL, PartyType.INDIVIDUAL)
+    const claimantResponseTestData = new ClaimantResponseTestData()
+
+    // as defendant
+    defendantResponseSteps.disputeAllClaim(testData, claimantResponseTestData)
+    I.see(testData.claimRef)
+    // check dashboard
+    I.click('My account')
+    I.see('You’ve rejected the claim. You need to tell us more about the claim.')
+    // check status
+    I.click(testData.claimRef)
+    I.see(testData.claimRef)
+    I.see('Claim status')
+    I.see('You’ve rejected the claim and said you don’t want to use mediation to solve it.')
+    I.click('Sign out')
+
+    // as claimant
+    userSteps.login(testData.claimantEmail)
+    claimantResponseSteps.viewClaimFromDashboard(testData.claimRef, false)
+    // check dashboard
+    I.click('My account')
+    I.see(testData.claimRef)
+    I.see(`${testData.defendantName} has rejected your claim.`)
+    // check status
+    I.click(testData.claimRef)
+    I.see(testData.claimRef)
+    I.see('Claim status')
+    I.see('The defendant has rejected your claim')
+    I.see(`They said they dispute your claim.`)
+    I.click('Sign out')
+  })
+
+Scenario(
+  'As a claimant I can reject the claim as I have paid less than the amount claimed @citizen @admissions @error',
```

Comment:
what is the @error do? is it ever used?

Reply:
Used @error where assertions are made on error state - so these should break when code fixed and the error tag removed.